ci-operator/step-registry/mtr/execute-ui-tests/mtr-execute-ui-tests-commands.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
What are your thoughts about adding another level to the file structure to separate tests from anything else that we are doing?

Create a `tests` directory under mtr so it would read `step-registry/mtr/tests/execute-ui-tests` and `step-registry/mtr/tests/execute-interop-ui-tests`

To me this gives others looking at a quick glance a nice separation of anything that is a test vs something that is just meant to complete configuration or just a tool to leverage.

Reply:
The only thing I have an issue with is the name of the refs being so repetitive: 'tests-execute-interop-ui-tests'

I guess we could do something like 'tests-ui' and 'tests-interop-ui'. @ascerra how do you feel about that?

Reply:
Resolved in commit [4602739](https://github.com/openshift/release/pull/36181/commits/4602739b55370b75da05c3fc8352787545510f44)